FuzzTesting/FuzzJSON.dict

```diff
@@ -0,0 +1,332 @@
+"\"mapUint64Bool\":"
+"\"mapUint64Bytes\":"
+"\"mapUint64Double\":"
+"\"mapUint64Fixed32\":"
+"\"mapUint64Fixed64\":"
+"\"mapUint64Float\":"
+"\"mapUint64Int32\":"
+"\"mapUint64Int64\":"
+"\"mapUint64Message\":"
+"\"mapUint64Sfixed32\":"
+"\"mapUint64Sfixed64\":"
+"\"mapUint64Sint32\":"
+"\"mapUint64Sint64\":"
+"\"mapUint64String\":"
+"\"mapUint64Uint32\":"
+"\"mapUint64Uint64\":"
+"\"oneofBool\":"
+"\"oneofBytes\":"
+"\"oneofDouble\":"
+"\"oneofEnum\":"
+"\"oneofFixed32\":"
+"\"oneofFixed64\":"
+"\"oneofFloat\":"
+"\"oneofInt32\":"
+"\"oneofInt64\":"
+"\"oneofMessage\":"
+"\"oneofSfixed32\":"
+"\"oneofSfixed64\":"
+"\"oneofSint32\":"
+"\"oneofSint64\":"
+"\"oneofString\":"
+"\"oneofUint32\":"
+"\"oneofUint64\":"
+"\"oneofgroup\":"
+"\"repeatedBoolExt\":"
+"\"repeatedBool\":"
+"\"repeatedBytesExt\":"
+"\"repeatedBytes\":"
+"\"repeatedDoubleExt\":"
+"\"repeatedDouble\":"
+"\"repeatedEnumExt\":"
+"\"repeatedEnum\":"
+"\"repeatedFixed32Ext\":"
+"\"repeatedFixed32\":"
+"\"repeatedFixed64Ext\":"
+"\"repeatedFixed64\":"
+"\"repeatedFloatExt\":"
+"\"repeatedFloat\":"
+"\"repeatedInt32Ext\":"
+"\"repeatedInt32\":"
+"\"repeatedInt64Ext\":"
+"\"repeatedInt64\":"
+"\"repeatedMessageExt\":"
+"\"repeatedMessageSet\":"
+"\"repeatedMessage\":"
+"\"repeatedSfixed32Ext\":"
+"\"repeatedSfixed32\":"
+"\"repeatedSfixed64Ext\":"
+"\"repeatedSfixed64\":"
+"\"repeatedSint32Ext\":"
+"\"repeatedSint32\":"
+"\"repeatedSint64Ext\":"
+"\"repeatedSint64\":"
+"\"repeatedStringExt\":"
+"\"repeatedString\":"
+"\"repeatedUint32Ext\":"
+"\"repeatedUint32\":"
+"\"repeatedUint64Ext\":"
+"\"repeatedUint64\":"
+"\"repeatedgroupExt\":"
+"\"repeatedgroup\":"
+"\"singularBoolExt\":"
+"\"singularBool\":"
+"\"singularBytesExt\":"
+"\"singularBytes\":"
+"\"singularDoubleExt\":"
+"\"singularDouble\":"
+"\"singularEnumExt\":"
+"\"singularEnum\":"
+"\"singularFixed32Ext\":"
+"\"singularFixed32\":"
+"\"singularFixed64Ext\":"
+"\"singularFixed64\":"
+"\"singularFloatExt\":"
+"\"singularFloat\":"
+"\"singularInt32Ext\":"
+"\"singularInt32\":"
+"\"singularInt64Ext\":"
+"\"singularInt64\":"
+"\"singularMessageExt\":"
+"\"singularMessageSet\":"
+"\"singularMessage\":"
+"\"singularSfixed32Ext\":"
+"\"singularSfixed32\":"
+"\"singularSfixed64Ext\":"
+"\"singularSfixed64\":"
+"\"singularSint32Ext\":"
+"\"singularSint32\":"
+"\"singularSint64Ext\":"
+"\"singularSint64\":"
+"\"singularStringExt\":"
+"\"singularString\":"
+"\"singularUint32Ext\":"
+"\"singularUint32\":"
+"\"singularUint64Ext\":"
+"\"singularUint64\":"
+"\"singulargroupExt\":"
+"\"singulargroup\":"
+"\"value\":"
+"\"wktAny\":"
+"\"wktApi\":"
+"\"wktBoolValue\":"
+"\"wktBytesValue\":"
+"\"wktDoubleValue\":"
+"\"wktDuration\":"
+"\"wktEmpty\":"
+"\"wktFieldMask\":"
+"\"wktFloatValue\":"
+"\"wktInt32Value\":"
+"\"wktInt64Value\":"
+"\"wktSourceContext\":"
+"\"wktStringValue\":"
+"\"wktStruct\":"
+"\"wktTimestamp\":"
+"\"wktType\":"
+"\"wktUint32Value\":"
+"\"wktUint64Value\":"
+
+# Some fragments
+"[]"
+"{}"
+"\"\""
+"''"
```

Comment:
AFAIK JSON doesn't accept single-quoted strings; should these be included? Likewise, `\x` and `\0` aren't valid in JSON, and `\/` isn't covered here.

Reply:
Removed single quote.

Removed `\x` and `\0`.

Added `\/`.